BT-Tracking/Interface/Help/HelpSearchVC.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  HelpSearchVC.swift
+//  BT-Tracking
+//
+//  Created by Lukáš Foldýna on 22/10/2020.
+//  Copyright © 2020 Covid19CZ. All rights reserved.
+//
+
+import UIKit
+
+private class HelpSearchItem: NSObject {
+    let article: HelpArticle
+    @objc dynamic let type: String
+    @objc dynamic let line: String
+
+    init(article: HelpArticle, type: String, line: String) {
+        self.article = article
+        self.type = type
+        self.line = line
+    }
+}
+
+final class HelpSearchVC: UITableViewController {
+
+    var articles: [HelpVM.Section] = [] {
+        didSet {
+            searchItems.removeAll()
+            for section in articles {
+                for article in section.items {
+                    searchItems.append(.init(article: article, type: "title", line: article.title))
```

Comment:
Zde bych misto type stringu zadefinoval ten type jako enum / struct / constantu, aby to Xcode mohl aspon type checknout.